unattended_installer/install_functions/opendistro/checks.sh

```diff
@@ -147,6 +158,27 @@ function checkArguments() {
 
 }
 
+function checkTools() {
+
+    # -------------- Check tools required to run the script (awk, sed, etc.) -----------------------------------------
+
+    toolList=(      "uname"
+                    "free"
+                    "tar"
+                    "awk"
+                    "free"
+                    "sed")
```

Comment:
Include `echo`, `cat`, `grep`, `rm`, `touch`, `kill`, `cp`, `mv`, `export`, `mkdir`, `curl`, `chown`, `sudo`, `chmod`, in the list. Then, don't exit in the first not found command. Instead of this, make a list of not-found dependencies and print the list, avoiding the re-run for every command.

Reply:
Done!

Reply:
Installation simulation, modifying the list of tools by adding the false commands 'badbad' and 'notsobad'.

`
sudo bash /tmp/unattended_installer/wazuh_install.sh --local --development --ignore-health-check --disable-spinner --all-in-one
`

```
03/02/2022 16:41:02 INFO: ---------------------------------- Missing tool -----------------------------------
03/02/2022 16:41:02 INFO: Missing tool report: badbad, notsobad,  are not installed. All this command's are necessary for the correct use of this tool.

```